lib/src/Uncertainty/Distribution/ComposedCopula.cxx

```diff
@@ -466,7 +466,33 @@ ComposedCopula::Implementation ComposedCopula::getMarginal(const Indices & indic
   if (!indices.check(dimension)) throw InvalidArgumentException(HERE) << "Error: the indices of a marginal distribution must be in the range [0, dim-1] and must be different";
   CopulaCollection marginalCopulas;
   const UnsignedInteger indicesSize = indices.getSize();
-  const UnsignedInteger size = copulaCollection_.getSize();
+  // Merge contiguous Gaussian copulas
```

Comment:
What does "contiguous" mean here?  I am confused since you check copulas at indices ii and size-1 below.

Reply:
The new copulaCollection variable stores the merged copulas. At the beginning it stores only the first copula from the original copulaCollection_ attribute. In the for loop I check if the next item in copulaCollection_ and the last item in copulaCollection can be merged into a new NormalCopula else I append the copulaCollection_[ii] into copulaCollection.

At the end we have one NormalCopula instance in copulaCollection for all sets of "contiguous" NormalCopula instances in copulaCollection_. For example [NormalC, NormalC, ClaytonC, NormalC, NormalC] leads to [NormalC, ClaytonC, NormalC].